src/docfx/lib/log/Log.cs

```diff
@@ -32,6 +32,19 @@ public static void Important(string message, ConsoleColor color)
             }
         }
 
+        public static void Error(Error error, ErrorLevel? level = null)
```

Comment:
think `ConsoleLog` is better than `Error`

Reply:
yep, initially I thought all console writes goes to `Log`, turned out to be not true